charms/istio-pilot/actions.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+set-tls-manually:
```

Comment:
I want this to not be confused with setting TLS with a TLS cert provider, though the workflow is different, and we can provide documentation, I can see confusion in our users. If you think otherwise, let me know.

Reply:
my vote is calling it just `set-tls` and explaining the nuances of it in the `documentation` field of the action.  Not that it is the only consideration, but I can't think of examples elsewhere that feel similar to including the `-manually` on this command.  But its up to you, I'm not blocking the PR on this

Reply:
It's fine, let's go with `set-tls` and have it very clear in the docs.